lib/src/view/tournament/tournament_screen.dart

```diff
@@ -366,74 +363,86 @@ class _FeaturedGame extends ConsumerWidget {
     return LayoutBuilder(
       builder: (context, constraints) {
         final boardSize = constraints.maxWidth;
-        switch (ref.watch(
-          tvControllerProvider(null, (featuredGame.id, featuredGame.orientation)),
-        )) {
-          case AsyncData(:final value):
-            {
-              final game = value.game;
-
-              final whitePlayer = _FeaturedGamePlayer(
-                game: game,
-                player: featuredGame.white,
-                side: Side.white,
-              );
-
-              final blackPlayer = _FeaturedGamePlayer(
-                game: game,
-                player: featuredGame.black,
-                side: Side.black,
-              );
-
-              return BoardThumbnail(
-                size: boardSize,
-                orientation: featuredGame.orientation,
-                fen: game.lastPosition.fen,
-                header: featuredGame.orientation == Side.white ? blackPlayer : whitePlayer,
-                footer: featuredGame.orientation == Side.white ? whitePlayer : blackPlayer,
-                lastMove: game.lastMove,
-              );
-            }
-          case _:
-            return BoardThumbnail(
-              size: boardSize,
-              fen: featuredGame.fen,
-              orientation: featuredGame.orientation,
-              header: const Shimmer(child: LoadingPlayerWidget()),
-              footer: const Shimmer(child: LoadingPlayerWidget()),
-            );
-        }
+
+        // Don't bother parsing the whole FEN here, we just need the turn
+        final turn = featuredGame.fen.endsWith(' b') ? Side.black : Side.white;
```

Comment:
@veloce Does chessground have some kind of convenience API for this? I didn't want to waste performance by parsing the whole string here

Reply:
Nope. That would be more a dartchess thing but for now there is only the Position API which parses the full fen. I don't think performance is an issue here though, so I wouldn't bother.

Reply:
Well for bullet games this might be called pretty often

Reply:
Fen parsing is cheap enough that even for real games (playing), the FEN is parsed each time the position change (it's a chessground parameter).

Reply:
Alright 👍